docs/source/notebooks/finch-usage.ipynb

```diff
@@ -18,8 +18,8 @@
     "import os\n",
     "from birdy import WPSClient\n",
     "#url = 'https://finch.crim.ca/wps'\n",
-    "url = 'https://pavics.ouranos.ca/twitcher/ows/proxy/finch/wps'\n",
-    "#url = 'http://localhost:5000'\n",
+    "#url = 'https://pavics.ouranos.ca/twitcher/ows/proxy/finch/wps'\n",
+    "url = 'http://localhost:5000'\n",
```

Comment:
Can not hardcode `localhost` like that.  Have to leave `pavics.ouranos.ca/twitcher/...` to not break Jenkins later (Jenkins runs the exact same notebook).

Get the wps url like so `url = os.environ.get("RAVEN_WPS_URL", "https://pavics.ouranos.ca/twitcher/ows/proxy/raven/wps")` (https://github.com/Ouranosinc/raven/blob/master/docs/source/notebooks/Delineation_workflow.ipynb) and in Travis override it to the localhost.  Probably Raven also have the Travis override example.

Reply:
done